### PR TITLE
Create join room screen

### PR DESCRIPTION
**Pull request by DanialBetres**
If you can think of a better name for the component pls let me know lol
also we should add react router right?

### CODE REVIEW

File: src/HomeScreen.tsx [hunk 1]
```diff
@@ -1,29 +1,54 @@
...
-import { VoteButton } from "shared";
+import { VoteButton } from 'shared';
 
 // TODO: smh think of a better name lol
-const BoxInput = ({title, iconClassname}: {title: string, iconClassname: string}) => (
-  <button className="btn home-screen__box-input" onClick={() => alert(title)}>
-      <div className={iconClassname}/>
-      <div className="type--large-white">{title}<span className="type--large-green">.</span></div> 
-    </button>
+const BoxInput = ({
```
Comment (donghee214) [inline src/HomeScreen.tsx]: What do you think about having BoxInput in its own file? So like maybe: 
feature/
    Home/
          Containers(Or screens idk)/
          Components/
                BoxInput.tsx

File: src/HomeScreen.tsx [hunk 2]
```diff
@@ -1,29 +1,54 @@
...
+      <span className="type--large-green">.</span>
+    </div>
+  </button>
 );
 
-const HomeScreen = () => (
+interface HomeScreen {
+  onClickHandler: (title: string) => boolean;
+}
+const HomeScreen = ({ onClickHandler }: HomeScreen) => (
```
Comment (donghee214) [inline src/HomeScreen.tsx]: So is homeScreen gonna be interacting with the data layer at all, or is this gonna be all presentational?

File: src/shared/components/CreateJoinRoomScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
+import React, { Component, Fragment } from 'react';
+
+import { PencilIcon } from '../../assets/images/pencilIcon';
+import { QuestionMarkIcon } from '../../assets/images/questionMarkIcon';
```
Comment (donghee214) [inline src/shared/components/CreateJoinRoomScreen.tsx]: Could we avoid relative pathing?

File: src/HomeScreen.tsx (hunk 1 above)
Comment (DanialBetres) [reply src/HomeScreen.tsx]: uhh If it's used only within homeScreen I'd prefer to leave it in here for two reasons: 

1. it's a small enough component
2. it's tightly coupled with the home screen component

If we're going to be using it elsewhere though then it voids my second reason and I think it makes sense to put it in it's own file. Are you strongly for putting it in it's own file?

File: src/HomeScreen.tsx (hunk 2 above)
Comment (DanialBetres) [reply src/HomeScreen.tsx]: hmmm I think we'll be dispatching an action to create a new room or join a new room from this component - so we will end up connecting it I think. Does that answer your question?

**Pull request merged**